contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -33,7 +33,7 @@ packages:
 - "ca-certificates"
 - "python3"
 remotes:
-- "url": "https://github.com/pivx-project/pivx.git"
+- "url": "https://github.com/mitchelvanamstel/SAPP_1_4"
```

Comment:
make it `https://github.com/sappcoin-com/SAPP` the current 1_4 repository will be renamed to the main one

Reply:
Changed both values to the SAPP repo's and the version